src/Router/RequiresAuth.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
```

Comment:
this import is not mandatory as of higher versions of React ! can check & remove.